src/flows/components/controlSearchBar.scss

```diff
@@ -0,0 +1,40 @@
+.cf-page-control-bar--fixed {
+  padding: 0px !important;
+}
+
+.flows-index--presetButtons {
+  animation: fadeIn 0.5s;
+  -webkit-animation: fadeIn 0.5s;
```

Comment:
i dont think you need these browser specific rules as the browser wars are over and any rubble that remains should be added by our css parser.